cg/apps/gt.py

```diff
@@ -37,3 +46,31 @@ def upload(self, bcf_path: str, samples_sex: dict, force: bool=False):
                 analysis_obj.sex = samples_sex[analysis_obj.sample_id]['analysis']
                 analysis_obj.sample.sex = samples_sex[analysis_obj.sample_id]['pedigree']
                 self.commit()
+
+    def get_trending(self, sample_id: str = '', days: str = '') -> dict:
+        """Get trending object with one or many samples."""
+        trending_call = self.base_call[:]
+
+        if sample_id:
+            trending_call.extend(['prepare-trending', '-s', sample_id])
+        elif days:
+            trending_call.extend(['prepare-trending', '-d', days])
+
+        try:
+            LOG.info('Running Genotype API to get data.')
+            output = subprocess.check_output(
+                ' '.join(trending_call),
```

Comment:
There is no need to join this. Subprocess does that for you.
```suggestion

```

Reply:
I dont get it to work when removing the join though. 

  File "/Users/mayabrandi/opt/cg/cg/meta/upload/vogue.py", line 18, in load_genotype
    samples = json.loads(samples)
  File "/Users/mayabrandi/anaconda/envs/cg/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/Users/mayabrandi/anaconda/envs/cg/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Users/mayabrandi/anaconda/envs/cg/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
(cg) 130-229-1-124-dhcp:cg mayabrandi$